lib/compile-typescript.js

```diff
@@ -13,8 +13,15 @@ module.exports = function compileTypescript(tsconfigFile, projectPath) {
     include: ['lib.*.d.ts']
   });
 
-  let ts = funnel(mergeTrees([libs, projectPath]), {
-    annotation: 'raw source'
+  let src = funnel(projectPath, {
```

Comment:
You should use two funnels, one for projectPath + `/src` and the other for projectPath + `/test`. We should not be forcing the root project path to be a watched dir (it will infinite loop rebuild if not using watchman).